llama-index-integrations/vector_stores/llama-index-vector-stores-chroma/tests/test_chromadb.py

```diff
@@ -29,20 +25,19 @@
     conn__ = chromadb.HttpClient(**PARAMS)  # type: ignore
     conn__.get_or_create_collection(COLLECTION_NAME)
 
-    chromadb_not_available = False
+    http_client_chromadb_mode = True
 except (ImportError, Exception):
-    chromadb_not_available = True
+    http_client_chromadb_mode = False
 
 
-@pytest.mark.skipif(chromadb_not_available, reason="chromadb is not available")
-def test_instance_creation_from_collection() -> None:
-    connection = chromadb.HttpClient(**PARAMS)
-    collection = connection.get_collection(COLLECTION_NAME)
-    store = ChromaVectorStore.from_collection(collection)
-    assert isinstance(store, ChromaVectorStore)
-
-
-@pytest.mark.skipif(chromadb_not_available, reason="chromadb is not available")
+# To test chromadb http-client functionality do:
+# cd tests
+# docker-compose up
+#
+@pytest.mark.skipif(
+    http_client_chromadb_mode == False,
```

Comment:
looks like linting doesnt like testing conditions like this, should be `http_client_chromadb_mode is False`

Reply:
fixed